### PR TITLE
Respond to Brian's comments

### LINKED ISSUE #221: Respond to reviewer comments
Respond to comments from Brian Coffey.

### LINKED ISSUE #223: Equation <-> Causal Graph compatibility
Currently, the causal graphs and the equations in section 2 do not match.

For instance:
- number of licensed drivers shows up in our causal graphs but not in our utility equations
- cross-bay shows up in our utility equations but not our causal graphs

We need to ensure our equations and our graphs match one another.
I think this is important for our credibility:
both in terms of us being diligent with our published materials and so that readers have confidence that we did what we claimed (even without having to check our code).

cc @hassan-obeid for viz.

@bouzaghrane:
1. will you edit the utility equations in Section 2 so they match exactly what we use in the utility equations for our simulations?  
For example, we use number of [autos per licensed driver](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L316) in the utility equations, not number of autos.

2. Then, will you edit the corresponding causal graphs so they:
- exactly include all "raw" variables that go into the utility equations
(e.g. [cross-bay](https://github.com/hassan-obeid/tr_b_causal_2020/blob/master/notebooks/final/7.0-mab-selection-on-observables-final.py#L327) and all other variables from that section of code)?

Please tag me on the PR after you're done so I can review!
Also, please point the PR to the `respond-to-brians-comments` branch and link it to this issue. 🙏🏽

### LINKED ISSUE #225: Make Sec 2 causal graphs 1 figure
Hey Amine,

Right now the causal graphs in section 2 are being split across pages.

Brian Coffey (who reviewed the chapter) pointed out that this isn't ideal, and it's also strange for readers who have to see the same graph two times in two different places.

Will you follow the example [here](https://tex.stackexchange.com/questions/196653/how-do-i-stack-two-figures-on-top-of-each-other-rather-than-side-to-side) to put Figure \ref{fig:DA_causal_2} through Figure \ref{fig:SR3_causal_2} into a single figure with sub-figures? You can keep the labels, they just need to go inside subfigure environments instead of in figure environments.

If you make a PR for this change, please point it at the `respond-to-brians-comments` branch and link it to this issue. 🙏🏽

### PR DESCRIPTION
**Pull request by timothyb0912**
This PR fixes #221.

It addresses all comments from Brian Coffey except for the inconsistency between our utility equations and causal graphs in section 2 (i.e. Issue #223).

### CODE REVIEW

**Comment by bouzaghrane** [comment]
I checked these and they look good to me. I also made edits directly here to the causal graph figure structure. Let me know if that's ok. @timothyb0912

**Comment by timothyb0912** [comment]
@bouzaghrane making the edits here is totally fine.

I pulled your changes and tried to run `tectonic article/main.tex` but it had an error.

Can you make sure the chapter compiles and push whatever changes are needed for that to happen?
Screen Shot 2021-02-07 at 5 49 16 PM

**Review by hassan-obeid — APPROVED** [review]
@timothyb0912 I went over the changes and all looks good to me. Let me know if there's a specific section you're unsure about.

**Comment by timothyb0912** [comment]
@hassan-obeid Nope, no particular section. I just want to make sure all of us see and are okay with everything that is being published.

**Comment by timothyb0912** [comment]
@bouzaghrane gentle ping on this. Please update the pdf, or let me know if you can't.

**Comment by bouzaghrane** [comment]
@timothyb0912 working on this now..for some reason the pdf doesn't compile, im trying to figure it out.

**Comment by timothyb0912** [comment]
Thanks @bouzaghrane -- I appreciate you!

**Comment by bouzaghrane** [comment]
@timothyb0912 I made changes and tried to generate the pdf, but for the life of me I can't figure out why I get that same error you do. If you look at the file changes tab, you'll see exactly the changes I made to the section and code responsible for generating the causal graphs. I am not sure why this error keeps popping up...it doesn't even make sense because it says that several references mentioned in the manuscript are undefined? when they clearly are? I wanted to still push changes and im still looking at where the problem could be.

**Comment by bouzaghrane** [comment]
@timothyb0912 I wanted to check in on this. Any idea why the pdf isn't compiling?

**Comment by timothyb0912** [comment]
@bouzaghrane I've been dealing with criminal, and medical, and too many other problems in my family at the moment.
Sorry for the delay brother. I haven't checked at all.

I imagine one needs to manually run the selection on observables notebook in addition to doing all the other steps from the makefile for the article compilation. There is a difference to how the notebook runs in the makefile vs when open manually which I exploit to make the tests run quickly on github CI vs having many simulations when run in the notebook to make a nice plot for the pdf.

File: article/sections/_2_graph_importance.tex [hunk 1]
```diff
@@ -104,31 +104,30 @@ \section{Perils of disregard}
 The goal of our simulation exercise is to show that ignoring the true generative model of the data (Figures \ref{fig:DA_causal_2} through \ref{fig:SR3_causal_2}) can result in arbitrarily biased treatment effects, even if the analyst knows the true outcome model (Equations \ref{eq:DA} through \ref{eq:SR3}).
 
 \begin{figure}
-   \centering
+\centering
+\begin{subfigure}
```
Comment (timothyb0912) [inline article/sections/_2_graph_importance.tex]: @bouzaghrane This was the breaking change for the pdf.

When using the subcaption package, we need to put the width commands next to `\begin{subfigure}` not next to `includegraphics`. See page 6: http://mirrors.ibiblio.org/CTAN/macros/latex/contrib/caption/subcaption.pdf

**Pull request merged**